include/odp/api/spec/init.h

```diff
@@ -402,6 +402,22 @@ int odp_term_abnormal(odp_instance_t instance, uint64_t flags, void *data);
  */
 void odp_log_thread_fn_set(odp_log_func_t func);
 
+/**
+ * Get current log function
+ *
+ * May be called before odp_init_global().
```

Comment:
Should this be unified with the newer PR (#2143): This one or "This function may be called before ODP initialization."

(Same for `odp_abort_fn_get()`.)

Reply:
Changed to "... if ODP is not initialized".